golang/cosmos/x/swingset/keeper/store_snapshotter.go

```diff
@@ -0,0 +1,601 @@
+package keeper
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/Agoric/agoric-sdk/golang/cosmos/vm"
+	"github.com/Agoric/agoric-sdk/golang/cosmos/x/swingset/types"
+	vstoragetypes "github.com/Agoric/agoric-sdk/golang/cosmos/x/vstorage/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+// ExportManifestFilename is the manifest filename which must be synchronized with the JS export/import tooling
```

Comment:
What is the nature of the synchronization? Should there be a specific cross-reference here?